deps/wazuh_testing/wazuh_testing/modules/logcollector/event_monitor.py

```diff
@@ -0,0 +1,165 @@
+import re
+
+from wazuh_testing import T_30, T_60
+from wazuh_testing.modules.logcollector import LOG_COLLECTOR_PREFIX, MACOS_LOG_COMMAND_PATH
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.tools import LOG_FILE_PATH
+
+
+def make_logcollector_callback(pattern, prefix=LOG_COLLECTOR_PREFIX, escape=False):
+    """Create a callback function from a text pattern.
+
+    It already contains the analsisd prefix.
```

Comment:
Change it to logcollector

Reply:
Done in [here]([40e7276](https://github.com/wazuh/wazuh-qa/pull/3486/commits/40e727636c5f3938a6430afe75851cecd436d81c))